promql/engine_test.go

```diff
@@ -751,7 +751,7 @@ load 10s
 			Interval: 5 * time.Second,
 		},
 		{
-			Query:       `count_values("wrong label!", metric)`,
+			Query:       `count_values("wrong label!\xff", metric)`,
```

Comment:
this test was relying on an error being thrown because the label name was invalid.  Under UTF-8 the old name is now valid, so the test was failing. Adding a bad unicode value causes it to once again throw an error.